sunpy/instr/aia/read_genx.py

```diff
@@ -0,0 +1,386 @@
+from __future__ import (absolute_import, division, print_function,
```

Comment:
would this file not be better named something like `read_aia_tables`? Because it's very specific to them rather than the genx files in general